plugins/onex/agents/configs/agent-registry.yaml

```
@@ -4,13 +4,13 @@
 # ONEX Agent Registry - Dynamic Role System
 # Version: 1.0.0
 # Purpose: Centralized registry for all agent identities and capabilities
-# Integration: agent-polymorphic-agent (Polly) dynamic transformation system
+# Integration: agent routing and dynamic transformation system
 
 schema_version: "1.0.0"
 mode: full
 agent_type: "registry"
 mode: "full"
```

YAMLlint (1.38.0)

[error] duplication of key "mode" in mapping

(key-duplicates)